package/koreader/package

```diff
@@ -2,15 +2,15 @@
 pkgname=koreader
 pkgdesc="KOReader"
 url=https://github.com/koreader/koreader
-pkgver=2020.08.01-1
-timestamp=2020-09-19T03:30Z
+pkgver=2020.09-1
+timestamp=2020-09-25T01:42Z
```

Comment:
This timestamp [should match the upstream release date](https://github.com/toltec-dev/toltec/blob/testing/docs/package.md#timestamp-required), i.e. `2020-09-17T01:52Z` if I’m not mistaken.

Reply:
Sure, it wasn't clear since the timestamp is previous to the last one.